sonic_platform_base/chassis_base.py

```diff
@@ -280,26 +280,64 @@ def get_module_index(self, module_name):
     # SmartSwitch methods
     ##############################################
 
-    def get_dpu_id(self, name):
+    def get_dpu_id(self, **kwargs):
```

Comment:
@oleksandrivantsiv since this is a base class can we make use `get_module_index()` ?

Reply:
`get_module_index` has a different meaning. It returns `An integer, the index of the ModuleBase object in the module_list`. `get_dpu_id` returns the physical ID of the DPU (its position)

Reply:
@oleksandrivantsiv can you rebase, looks like this API already exist https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/chassis_base.py#L283

Reply:
Yes, the `get_dpu_id` already exists. I changed the parameter from `name` to `**kwargs`. This API will have a different meaning for switch and DPU. Please check the description